README.md

```diff
@@ -695,7 +695,7 @@ You can also use different in and out transitions for the same slide:
     And it starts again.
 </section>
 ```
-
+You can choose from `none` `fade` `slide` `convex` `concave` `zoom`.
```

Comment:
"``You can choose from `none`, `fade`, `slide`, `convex`, `concave`, and `zoom`.``"?

Reply:
Sorry, I forgot to fill in the description. I couldn't find a list available transitions, so I was hoping to get them enumerated in the docs somewhere.

Reply:
Sorry I was not clear in my comment: I was trying to suggest adding commas and a final conjunction ("and") to the new sentence. :wink: